server/server/src/sync_on_central/mod.rs

```diff
@@ -47,3 +51,13 @@ async fn push(
 
     Ok(web::Json(response))
 }
+
+#[post("/sync/site_status")]
```

Comment:
Maybe should be a GET request, at least it is in v5... but we had the pattern set up for passing the sync settings in the POST request body so just stuck with that?

Reply:
I think you can, but you're not really meant to/it's not meant to have any impact?? I was just meaning that for v5, we send the site id and other sync settings as request headers, but we have the pattern in v6 of sending them as the POST body. `site_status` sort of sounds like the kind of thing you would `get`, but I think it's better to have the consistent mechanism to send the sync settings, so will stick with the POST 🤷‍♀️